pwiz_tools/Skyline/Model/Hibernate/Formats.cs

```diff
@@ -40,6 +40,8 @@ public static class Formats
         public const string Mz = "0.####";
         public const string SamplingTime = "0.00";
         public const string OneOverK0 = "0.####";
+
+        public const string IonMobility = "0.#####";

```

Comment:
This is probably more digits than needed - perhaps rename OneOverK0 to IonMobility instead